app/controllers/users_controller.rb

```diff
@@ -1,5 +1,12 @@
 class UsersController < ApplicationController
-  before_action :load_user, only: :show
+  before_action :logged_in_user, except: %i(show new create)
+  before_action :logged_in_user, except: %i(show new create)
```

Comment:
delete this line because you defined it above

Reply:
I have deleted it before create this pull request.